minkindr/include/kindr/minimal/implementation/angle-axis-inl.h

```diff
@@ -62,6 +62,15 @@ AngleAxisTemplate<Scalar>::AngleAxisTemplate(const RotationMatrix& matrix) :
   // \todo furgalep Check that the matrix was good...
 }
 
+template<typename Scalar>
+AngleAxisTemplate<Scalar>::AngleAxisTemplate(const Vector3& angleAxis) {
+    Scalar norm = angleAxis.norm();
+    CHECK_LE(1e-8, norm);
```

Comment:
I would turn it around, i.e. `CHECK_GT(norm, 1e-8)`, is it realistic that this never occurs? should we just warn and set a random unit axis?